MST-348 Add the verification data detail modal so support can see those info

schenedx opened the pull request:
@staubina and @edx/masters-devs-cosmonauts Please review
Add a button and a model to the ID Verification panel to show list of ID verification for the user
See screenshot:

<img width="1313" alt="Screen Shot 2020-08-26 at 3 31 48 PM" src="https://user-images.githubusercontent.com/16839373/91348185-6e102180-e7b1-11ea-81a4-76056ed27cdd.png">

<img width="1313" alt="Screen Shot 2020-08-26 at 3 32 45 PM" src="https://user-images.githubusercontent.com/16839373/91348197-71a3a880-e7b1-11ea-90fd-4c5f1e9486b5.png">

staubina commented on src/users/UserSummary.jsx:
```diff
@@ -123,7 +149,31 @@ export default function UserSummary({
+
+  // Modal to display extra data for Idv records
+  const openIDVModal = (title, data) => {
+    setDetailIdvData(data);
```
The expiration date time that is in the modal is not formatted into a readable format. It will help keep the UI consistent if we run the date format on it. This might be easily solved by just moving where/when the date is formatted. 

It looks like you already format the date for the summary block.

staubina commented:
FYI @kashifch, I wanted you to know this was an upcoming change.

alangsto commented on src/users/UserSummary.jsx:
```diff
@@ -123,7 +149,31 @@ export default function UserSummary({
-          onClick={() => openModal(result.provider, result.extraData)}
+          onClick={() => openSSOModal(result.provider, result.extraData)}
+        >
+          Show
```
NIT: This is really small, but the padding around this element looks different, making it seem like "Show" isn't in line with the other columns.

schenedx commented:
@staubina and @alangsto Addressed your feedbacks.
The modal dialog being too small is not a problem I am going to solve in this PR. The Modal is a paragon component. In order to make the modal bigger, we need to bump up the version of Paragon. That breaks all the buttons and collapsible components on the page. I have a fix there but that's just too complicated for this PR. 
I'll find a way to solve that later.
